Tools/Scripts/git-webkit

```diff
@@ -50,7 +50,7 @@ def additional_setup(args, repository):
     if not isinstance(repository, local.Git):
         return 0
 
-    log.warning('Setting merging behavior for changelogs...')
+    log.info('Setting merging behavior for changelogs...')
```

Comment:
would it prevent it from being printed in regular `git-webkit setup` workflow?

Reply:
Yes! I forgot to change this in a change a few weeks back.